tests/NSubstitute.Acceptance.Specs/ReceivedCalls.cs

```diff
@@ -1,6 +1,7 @@
 using NSubstitute.Exceptions;
 using NSubstitute.ReceivedExtensions;
 using NUnit.Framework;
+using NUnit.Framework.Legacy;
```

Comment:
unused?

Reply:
this is nbeeded for StringAssert

Reply:
yes, this related to `Update nuget packages for test projects`